Shared/Networking/ConnectionInfo.swift

```diff
@@ -210,32 +210,34 @@ public class ConnectionInfo: Codable {
         Current.Log.error(errMsg)
 
         #if os(iOS)
-        let alert = UIAlertController(title: "URL Unavailable",
-                                      // swiftlint:disable:next line_length
-                                      message: "Expected to have a \(self.activeURLType) but none available! Please enter the URL. App will exit after entry, please reopen.",
-                                      preferredStyle: .alert)
-
-        var textField: UITextField?
-
-        alert.addTextField { (pTextField) in
-            pTextField.placeholder = self.activeURLType.description
-            pTextField.clearButtonMode = .whileEditing
-            pTextField.borderStyle = .none
-            textField = pTextField
-        }
+        DispatchQueue.main.async {
+            let alert = UIAlertController(title: "URL Unavailable",
+                                          // swiftlint:disable:next line_length
+                                          message: "Expected to have a \(self.activeURLType) but none available! Please enter the URL. App will exit after entry, please reopen.",
+                                          preferredStyle: .alert)
```

Comment:
Should we put these strings into Lokalise?

Reply:
yeah, probably - but let's maybe take a look at how we deal with this alongside websocket work so we can maybe avoid needing to care, too